src/components/CartItems.jsx

```diff
@@ -0,0 +1,60 @@
+export const CartItems = () => {
+  return (
+    <div className="cart-items">
+      <div className="card-horizontal pos-relative flex m-1">
```

Comment:
Extract out the card as a separate component and used `Array.map` to render multiple cards

Reply:
I haven't worked on the cart functionality. So, have just kept placeholder cards for now.